airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_tracker/TrackingMetadata.java

```diff
@@ -96,6 +96,7 @@
           final JobOutput jobOutput = lastAttempt.getOutput().get();
           if (jobOutput.getSync() != null) {
             final StandardSyncSummary syncSummary = jobOutput.getSync().getStandardSyncSummary();
+            metadata.put("sync_start_time", syncSummary.getStartTime());
```

Comment:
should we also add end time?

Reply:
duration is computed using the end time and we now have start time so we can always re-compute the reverse.
Let's leave it out to make the amount of data transiting through segment a little smaller